### PR TITLE
Follow changes on Nexus API

### PR DESCRIPTION
**Pull request by csillag**
This has the auto-generated code changes from #1121, plus our fixes.

### CODE REVIEW

**Review by lubej — APPROVED** [review]

File: src/oasis-nexus/generated/api.ts [hunk 1]
```diff
@@ -1555,10 +1562,28 @@ export const ConsensusEventType = {
...
   /** The module of a failed transaction. */
   module?: string;
+  /** The error parameters, as decoded using the contract abi. Present only when
+- the error originated from within a smart contract (e.g. via `revert` in Solidity), and 
+- the contract is verified or the revert reason is a plain String.
+If this field is present, `message` will include the name of the error, e.g. 'InsufficentBalance'.
+Note that users should be cautious when evaluating error data since the
+data origin is not tracked and error information can be faked.
+ */
+  revert_params?: EvmAbiParam[];
```
Comment (lukaw3d) [inline src/oasis-nexus/generated/api.ts:1586]: Diff doesn't make it clear what all the new fields are. We should extract a list of new fields to determine which ones we should start using in existing code

Comment (lukaw3d) [reply src/oasis-nexus/generated/api.ts:1586]: Split in https://github.com/oasisprotocol/explorer/pull/1190/commits/cd0bf54c7b5f7441cda2cdb03be1161e7e290d9d

Comment (lukaw3d) [reply src/oasis-nexus/generated/api.ts:1586]: Split in https://github.com/oasisprotocol/explorer/pull/1190/commits/86ff7f729ec986388df07f76cb97132273751f00

Comment (lukaw3d) [reply src/oasis-nexus/generated/api.ts:1586]: https://github.com/oasisprotocol/explorer/issues/1222

**Pull request merged**